### ISSUE TITLE
SVG images cause background to have vertical offset in Safari

### ISSUE THREAD

**Issue by rbirkby** [gnab/remark#339]
Adding an SVG image to markdown as an image causes any background image to be positioned incorrectly in Safari/PhantomJS/Webkit. Chrome renders OK.

```markdown
background-image: url(images/background.png)

# Slide with SVG

The background has a vertical offset

![](images/drawing.svg)
```

## Example output

Without SVG:

![image](https://cloud.githubusercontent.com/assets/203034/15674545/459442fe-2735-11e6-9c6d-68d9aaf7d1f3.png)

With SVG:

![image](https://cloud.githubusercontent.com/assets/203034/15674554/541ccb0c-2735-11e6-9754-5f4244d8dc76.png)

## Testcase

https://github.com/rbirkby/DeckTapeIssue

## Probable cause

Originally believed to be a Decktape-only issue, https://github.com/astefanutti/decktape/issues/52,  but seems to be a Safari/Webkit issue. @astefanutti pointed out a missing background-position in webkit vs blink.

**Comment by gnab** [comment]
In version 0.14.0 the `background-position` can be overridden, which should make this issue fixable as mentioned in #52.

**Comment by rbirkby** [comment]
I don't see a PR request to fix this issue in Safari? Is RemarkJS still broken in Safari?

**Comment by gnab** [comment]
Seems I got the linking wrong, didn't realize it was in the DeckTape project.

[This pull-request](https://github.com/gnab/remark/commit/d0332282a6f9f5be7f41c424b8055af6f6e93d92) included in 0.14.0 enables overriding `background-position`, so you can hopefully set the missing `background-position` mentioned in your original post.

**Comment by rbirkby** [comment]
Apologies - I'm not being clear enough.
There is a bug in RemarkJS when used with Safari and SVG which prevents Safari from displaying RemarkJS content correctly. This bug needs fixing in RemarkJS core. It is not a user customization for downstream tooling.

Unless RemarkJS specifically does not support Safari?
